frontend/src/lib/components/CommandBar/constants.ts

```diff
@@ -14,9 +14,16 @@ export enum Tab {
     Insight = 'insight',
     Notebook = 'notebook',
     Person = 'person',
+    Group0 = 'group_0',
+    Group1 = 'group_1',
+    Group2 = 'group_2',
+    Group3 = 'group_3',
+    Group4 = 'group_4',
```

Comment:
why five groups?

Reply:
This seems like the biggest point of confusion / contention here. Is there any means searching for all groups at once and categorising later? We seem add a lot of complexity to the `searchBarLogic` to support this

Reply:
The search needs to go through 5 different endpoints (one for each possible group). This is also how it works currently. I thought about using an abstraction, but think the verbose is more easily understandable for the future traveller. Note: I'm thinking about adding a query cache in a follow-up, then this might be changed as well.

Reply:
> why five groups?

(Ignore if you already know this, adding context just in case). PostHog supports up to 5 groups for each project. In the local demo project these are "Organization", "Account", "Instance", "Customer" and "Project" for example. Other groups will be useful for different customers and so we just have "Group 0,1,2,3,4" in our data model. This is the `group_type_index`.
The endpoint we (re-)use here has a mandatory `group_type_index`, thus 5 calls. Also these refer to different entities with different tabs, so a reducer per group is useful anyway.

Reply:
Gotcha! Thanks for the context. Tbh it sounds like a bit of a legacy mess that should just have been a "has-many" relationship 😅 What if I want six groups?

Your approach totally makes sense then